README.md

```diff
@@ -5,4 +5,14 @@
 A guide to the installation and configuration of this SDK can be found here: [https://docs.silamoney.com/docs/python-sdk](https://docs.silamoney.com/docs/python-sdk)
 
 ### Endpoint Methods
-Example code for each SDK endpoint method is found in the associated endpoint's section in our docs, which can be found here: [https://docs.silamoney.com/docs/get-started](https://docs.silamoney.com/docs/get-started)
\ No newline at end of file
+Example code for each SDK endpoint method is found in the associated endpoint's section in our docs, which can be found here: [https://docs.silamoney.com/docs/get-started](https://docs.silamoney.com/docs/get-started)
+
+### Debug Logging
+
+If you'd like to set additional debug logging, then make sure to set the logging level when you run your code:
```

Comment:
Add a note here mentioning that if you don't call `logging.basicConfig`, the logging level is set to `WARNING` by default, and so the `logger.debug()` messages will not be logged to the console.